esx_service/cli/vmdkops_admin.py

```diff
@@ -857,8 +859,8 @@ def tenant_rm(args):
     # If args "remove_volumes" is not specified in CLI
     # args.remove_volumes will be None
     if args.remove_volumes:
-        if args.remove_volumes == 'True':
-            remove_volumes = True
+        print "All Volumes will be removed"
```

Comment:
this is not Python v3 compatible. Please use function `print("msg")`, not language construct `print msg`